boot.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
-source venv/bin/activate
+
+# conditionally activate virtualenv, since the docker container does not need it
```

Comment:
why do you feel like that venv is not needed. Its not really slowing anything down / costing anything and there are at least some reasons to use them besides the fact that it has proven to work very well over many years ? 

https://vsupalov.com/virtualenv-in-docker/#:~:text=Docker%20is%20great%20at%20packaging,Docker%20image%20without%20using%20virtualenv.

Reply:
I personally feel like it's an anti pattern that unnecessarily increases the container size, given that docker is already a tool for dependency management. This change could be removed if you believe it is better to do so though.

Reply:
ok thanks for your feedback on this, I will focus a bit on vue3 first and do some further research at some point. If anyone from the community wants to state their opinion feel free.

Reply:
then lets split the venv removal into another PR, so this PR can be pushed